cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1767,6 +1770,27 @@ def default_for(
                 "[runtime][<namespace>][events]submission timeout"
             )
         ))
+    with Conf('CLI', desc='''
+        Configure command line interface options.
+
+        .. versionadded:: 8.1.0
+    '''):
+        Conf('communication method',
+             VDR.V_STRING, 'zmq',
+             options=[meth.value for meth in CLICommsMeth], desc='''
+            The communication method used to interact with a running workflow
+            via the command line interface.
+
+            Options:
+
+            zmq
+               Direct client-server TCP communication via network ports
+            http
+               Via the running Hub proxy and/or UI-Server (requires UI
+               Server installation)
+
+            .. versionchanged:: 8.1.0
+        ''')
```

Comment:
I think we might use `[platforms][localhost]communication method` for this configuration, see https://github.com/cylc/cylc-flow/issues/5265

Reply:
Ok, wasn't sure if that was job specific or not.